internal/commands/login/inputs.go

```diff
@@ -0,0 +1,51 @@
+package login
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	inputFieldPublicAPIKey  = "publicAPIKey"
+	inputFieldPrivateAPIKey = "privateAPIKey"
+)
+
+type inputs struct {
```

Comment:
I spent some time thinking about this inputs struct and how it would grow/if we wanted to nest the API keys here. But this approach makes sense to me. I don't imagine this growing too massive so I think the resolve method, where we go one by one into the input fields and add a question, is a fine solution.

Reply:
yeah... I'm toying around with a few ideas in the `user create` ticket I have in progress right now about this.  let's revisit this stuff on that PR